src/components/emotes.ts

```diff
@@ -0,0 +1,8 @@
+import { Emoji } from 'discord.js';
+import { CommandoClient } from 'discord.js-commando';
+
+export const emoteList: { [key: string]: Emoji } = {};
+
+export const initEmotes = (client: CommandoClient): void => {
+  client.emojis.cache.forEach((emoji) => (emoteList[emoji.name] = emoji));
```

Comment:
what emojis are in this cache? is it all emojis in every server that the bot has access to?

Reply:
yeah, this way I was thinking we could always give a backup emote server for codey if needed.